src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -244,24 +243,40 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           {...scheduleNGWithAttendees}
         />
       </div>
-      {!isEventsLoading && (
-        <div className="NavBarWrapper__download-buttons">
-          {isShowPersonalDownloadBtn && (
+      <div className="NavBarSchedule__download-buttons">
+        {!isEventsLoading && (
+          <>
+            {isShowPersonalDownloadBtn && (
+              <Button
+                onClick={downloadPersonalEventsCalendar}
+                customClass="NavBarSchedule__download-schedule-btn"
+              >
+                Download your schedule
+              </Button>
+            )}
             <Button
-              onClick={downloadPersonalEventsCalendar}
-              customClass="NavBarWrapper__download-schedule-btn"
+              onClick={downloadAllEventsCalendar}
+              customClass="NavBarSchedule__download-schedule-btn"
             >
-              Download your schedule
+              Download full schedule
```

Comment:
Git shows the diff wrong. The change was nesting under an empty element, indentation and replacing `NavBarWrapper` class names with `avBarSchedule`